HTML - 38/memorizeTheOrderOfRedCards.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Red cards</title>
+</head>
+<body>
+<table style="border: 1px solid black; border-spacing: 5px;">
+    <tr>
+        <td id="0" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></td>
+        <th id="1" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="2" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="3" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="4" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>

```

Comment:
- I used here only numbers because I can just turn them into int. If it has letters I can't use it in wanted way. What is "the dom" in your second sentence? Also, data-id is what? And el.dataset.id?
- I have never used eventListener, will try and let you know how it went.
- Okay